opencensus/trace/internal/span_test.cc

```diff
@@ -174,6 +175,18 @@ TEST(SpanTest, FullSpanTest) {
   span.AddAttribute("key2", "value2");
   span.AddAttributes({{"key3", "value3"}, {"key4", 123}, {"key5", false}});
 
+  // String (as opposed to literal).
+  std::string val = "value";
+  val += "6";
+  span.AddAttribute("key6", val);
+
+  // Test that StrCat's output outlives the AddAttribute call.
+  span.AddAttribute(absl::StrCat("ke", "y7"), absl::StrCat("val", "ue7"));
+  span.AddAttributes(
+      {{absl::StrCat("key", "10"), "value10"},
```

Comment:
Please also test that the values are there 👍

Reply:
Done. Thanks!